helm/kubeless/templates/zk-stateful.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: apps/v1beta1
+kind: StatefulSet
+metadata:
+  name: {{ template "fullname" . }}-zoo
+  labels:
+{{ include "labels.standard" . | indent 4 }}
+spec:
+  serviceName: "{{ template "fullname" . }}-zoo"
+  replicas: {{ .Values.zookeeper.statefulSet.replicaCount }}
+  template:
+    metadata:
+      labels:
+        kubeless: {{ template "fullname" . }}-zookeeper
+    spec:
+      terminationGracePeriodSeconds: 10
+      containers:
+        - name: zookeeper
+          image: "{{ .Values.zookeeper.statefulSet.image.repository }}:{{ .Values.zookeeper.statefulSet.image.tag }}"
+          imagePullPolicy: {{ .Values.zookeeper.statefulSet.image.pullPolicy }}
+          env:
+            - name: ZOO_SERVERS
+              value: server.1=zoo-0.zoo:2888:3888:participant
+          ports:
+            - containerPort: 2181
+              name: client
+            - containerPort: 2888
+              name: peer
+            - containerPort: 3888
+              name: leader-election
+          volumeMounts:
+            - name: zookeeper
+              mountPath: /bitnami/zookeeper
+            # There's defaults in this folder, such as logging config
```

Comment:
It is there https://github.com/kubeless/kubeless/blob/master/manifests/zookeeper/zk-stateful.yaml#L31-L38